src/extension.ts

```diff
@@ -13,6 +13,17 @@ import { UI } from './vscode/UI'
 import { log, toLog } from './vscode/Log'
 import { getHoverProvider } from './vscode/providers/Hover'
 
+// Word separator characters for CommonLisp.
+// These determine how a double-click will extend over a symbol.
+// Instead of defining what is contained in a symbol,
+// VSCode defines what is NOT in a symbol (or whatever).
+//
+// Using current (2023-04-27) version of config item minus hyphen.
+// Could also get() the default setting at load time and remove the hyphen
+// but it seems like a good idea to take control of this configuration item
+// in order to be independent of any future changes to the default setting.
+const wordSeparators = "`~!@#$%^&*()=+[{]}\\|;:'\",.<>/?"
```

Comment:
The joy of lisp, most of these aren't separators, They're valid in variable names. I think it's just these,

    const wordSeparators = "`|;:'\","

Reply:
Oops, forgot to include the parens,

    const wordSeparators = "`|;:'\",()"

Reply:
Done and pushed.